geniza/footnotes/admin.py

```diff
@@ -149,22 +148,16 @@ def get_queryset(self, request):
         return (
             super()
             .get_queryset(request)
+            .metadata_prefetch()
+            .footnote_count()
```

Comment:
very nice. Is `footnote_count` not inside `metadata_prefetch` because in the event we have a queryset (from admin) we don't need to prefetch but we do need to footnote_count?

Reply:
yes, that was my thought — they aren't necessarily related and we could easily want one and not the other